14-EventToCommandBehavior/src/PrismSample/Views/EventArgsConverterExamplePage.xaml

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="PrismSample.Views.EventArgsConverterExamplePage"
+             Title="Event Args Converter Example"
+             
```

Comment:
remove the empty line and whitespace here

Reply:
Done with the latest commit. :)